Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -236,7 +238,7 @@ - (NSDictionary *)liveContext
     context[@"network"] = ({
         NSMutableDictionary *network = [[NSMutableDictionary alloc] init];
 
-        if (self.bluetooth.hasKnownState)
+        if (self.bluetooth && self.bluetooth.hasKnownState)
```

Comment:
Yeah I was considering not adding the `self.bluetooth` check - I don't have a strong preference either way so I ended up adding it.